sdk/storage/storage-blob-change-feed/review/storage-blob-change-feed.api.md

```diff
@@ -0,0 +1,61 @@
+## API Report File for "@azure/storage-blob-change-feed"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { BlobServiceClient } from '@azure/storage-blob';
+import { PagedAsyncIterableIterator } from '@azure/core-paging';
+
+// @public (undocumented)
+export class BlobChangeFeedClient {
+    constructor(blobServiceClient: BlobServiceClient);
+    // (undocumented)
+    getChanges(options?: ChangeFeedGetChangesOptions): PagedAsyncIterableIterator<BlobChangeFeedEvent, BlobChangeFeedEventPage>;
+    }
+
+// @public (undocumented)
+export interface BlobChangeFeedEvent {
+    // Warning: (ae-forgotten-export) The symbol "BlobChangeFeedEventData" needs to be exported by the entry point index.d.ts
```

Comment:
Do we have swagger for change feed feature as well? or something is missed to be exported?

Reply:
No swagger. Fixed by exporting the `BlobChangeFeedEventData`.